htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -201,6 +201,7 @@ class FactureFournisseur extends CommonInvoice
 
 	// Warning: Do not set default value into property defintion. it must stay null.
 	// For example to avoid to have substition done when object is generic and not yet defined.
+	public $amount_ht;
```

Comment:
We alread have total_ht for amount. May be the problem is in the code rhat use amount_ht ? What will be the difference between amount_ht and total_ht ?